docs/40_node-operation/10_getting-started/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Getting Started
+---
+
+<head>
+    <title>Getting Started with Node Operation</title>
+</head>
+
+This section digs into the basic knowledge you need to run a node. All node operators (apis, producers, history nodes)
+should know the information in this section.
```

Comment:
should know -> should become familar with (or should get acquainted with)

(sounds more inviting)